pkg/commands/run_test.go

```diff
@@ -118,13 +182,41 @@ const expectedS3ScanResult = `{
             }
           }
         },
+        {
```

Comment:
Strange that this check didn't work before, but according to the [s3onlycache.json](https://github.com/aquasecurity/trivy-aws/blob/main/pkg/commands/testdata/s3onlycache.json#L192-L229)  cache file the bucket has no logging.